TUM Campus App/PlanDetailsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PlanDetailsViewController.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 16.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class PlanDetailsViewController: UIViewController, UIScrollViewDelegate {
+
+    @IBOutlet weak var scrollView: UIScrollView!
+    var imageView: UIImageView!
```

Comment:
Yes I use it more than once: 
https://github.com/TCA-Team/iOS/pull/50/files#diff-eb09460ec53bb01303365b20ed9ba5b1R47